content/en/docs/faas/lambda-auto-instrument.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Lambda Auto-Instrumentation
+linkTitle: Lambda Auto-Instrumentation
+weight: 11
+description:
+  Automatically instrument your Lambdas with OpenTelemetry 
+---
+
+The OpenTelemetry community provides standalone instrumentation Lambda layers for the following languages:
+
+* Java
+* JavaScript
+* Python
+
+These can be added to your Lambda using the AWS portal to automatically instrument your application. These layers do not include the Collector which is an optional but recommended standalone Lambda layer.
+
+### Language Requirements
+
+<!-- prettier-ignore -->
+{{< tabpane text=true >}}
+{{% tab Java %}}
+
+The Lambda layer supports the Java 11 (Corretto) Lambda runtime. It does not support the Java 8 Lambda runtimes. For more information about supported Java versions, see the OpenTelemetry Java documentation.
+
+Note: ADOT Lambda Layer for Java Auto-instrumentation Agent - Automatic instrumentation has a notable impact on startup time on AWS Lambda and you will generally need to use this along with provisioned concurrency and warmup requests to serve production requests without causing timeouts on initial requests while it initializes.
```

Comment:
Fix name?

Reply:
That's covered under tips below step 5. Should we bring it up here?

Reply:
Oops, missed that... Maybe just add a `(see tips section below for ways to improve startup time.)`
Alternatively, I do think it would make sense to move that to the java specific section since it doesn't apply for other languages.